tests/flow/test_intern_string.py

```
@@ -0,0 +1,426 @@
+        # query should fail
+        q = """MATCH (n)
+               SET n.v = 'new-string'
+               WITH n
+               SET n.x = n
+               RETURN n.v, typeof(n.v)"""
+
+        try:
+            # expecting query to fail
+            res = self.graph.query(q)
+            self.env.assertFalse("query should fail")
+        except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)